cvat/apps/tf_annotation/views.py

```diff
@@ -236,8 +236,7 @@ def create_thread(tid, labels_mapping):
         # Get job indexes and segment length
         db_task = TaskModel.objects.get(pk=tid)
         db_segments = list(db_task.segment_set.prefetch_related('job_set').all())
-        segment_length = max(db_segments[0].stop_frame - db_segments[0].start_frame + 1, 1)
-        job_indexes = [segment.job_set.first().id for segment in db_segments]
+        max_shape_id = max(segment.job_set.first().max_shape_id for segment in db_segments)
```

Comment:
Max shape id in this case is -1 because we will clear task before save. 
Consequently we don't need compute and sent it to convert_to_cvat_format as an argument.